config/environments/development.rb

```diff
@@ -17,7 +17,8 @@
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = true
-  config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
+  Rails.application.routes.default_url_options =
+    config.action_mailer.default_url_options = { host: "localhost", port: 3000 }
```

Comment:
We'll need to watch this when merging into the other forks, and update the production environments. It looks like url helpers are defaulting to `example.com`.